package.json

```diff
@@ -251,8 +271,30 @@
         {
           "command": "TabNine::assistantToggle",
           "when": "tabnine-assistant:capability"
-        }
-      ]
+        },
+        {
+					"command": "Tabnine.hideSuggestion",
```

Comment:
why do you need this declaration here?

Reply:
so it does not appear in the command pallete?

Saw that's what they did in the comments api sample

Reply:
i think you can remove it from here